.travis.yml

```diff
@@ -150,9 +158,26 @@ after_success:
 after_failure:
 - dmesg | tail -100
 after_script:
-- if [ $TRAVIS_SECURE_ENV_VARS == true ]; then
+- if [ $TRAVIS_SECURE_ENV_VARS == true && $TRAVIS_EVENT_TYPE != cron ]; then
     $GCLOUD/gcloud components -q update gsutil;
     REPORT_PATH=$TRAVIS_BRANCH_$TRAVIS_JOB_NUMBER;
     $GCLOUD/gsutil -m cp -z html -z js -z xml -z css -r build/reports/tests gs://hellbender/test/build_reports/$REPORT_PATH/;
     echo "See the test report at https://storage.googleapis.com/hellbender/test/build_reports/$REPORT_PATH/tests/test/index.html";
   fi
+# This creates and uploads the gatk zip file to the nightly build bucket, only keeping the 10 newest entries
```

Comment:
This is a good point, I originally had it there but I was running into issues getting the code to install gsutil . Its notable however that even if ruby fails to install in the install code but it passes all the subsequent tests (none in this case so its a low bar) it will consider the execution a success and still run the after-success script. I will move it back there because I fixed the issue that was giving me a headache and it might catch some other issue that would pop up.